examples/step-98/step-98.cc

```diff
@@ -0,0 +1,974 @@
+            // Submit these gradients for integration (standard Laplace weak
+            // form).
+            for (unsigned int q = 0; q < phi.n_q_points; ++q)
+              phi.submit_gradient(phi.get_gradient(q), q);
+            // Perform the integration (test function gradients against
+            // submitted gradients).
+            phi.integrate(EvaluationFlags::gradients);
+            // The results of the local operator application (A_cell * u_cell)
+            // are now stored within each 'phi' object.
+          }
+
+        // Gather the results of the local operator application from the cells
+        // into the 'local_residual' vector for the patch.
+        // 'true' indicates accumulation is needed if multiple cell DoFs map
+        // to the same patch DoF (though typically not the case here).
+        patch_eval.gather_local_to_patch(ArrayView<double>(local_residual),
+                                         true);
+
+        // --- Step 3: Compute the actual residual on the patch ---
+        // residual = rhs - A*solution (Note: local_residual currently holds
+        // A*solution)
+        for (unsigned int i = 0; i < local_residual.size(); ++i)
+          local_residual[i] = local_rhs[i] - local_residual[i];
+
+
+        // --- Step 4: Apply the inverse patch operator ---
+        // Solve the local system A_patch * correction = residual for the
+        // correction. 'inverse' is the PatchTensorInverse object initialized
+        // earlier.
+
+        patch_inverse_operator.apply_inverse(
+          ArrayView<number>(local_correction.data(), n_dofs_interior),
+          ArrayView<const number>(local_residual.data(), n_dofs_interior));
+
+        // inverse.vmult(local_correction, local_residual);
+
+        // --- Step 5: Update the solution ---
+        // The standard smoother update is: u_new = u_old + correction
+        // However, the distribute functions from underlying FEEvaluations
+        // perform an *addition* to the target vector. Since there are DoF that
+        // are shared on multiple cells and we want to add the correction only
+        // once, we need zero out the "duplicates".
+        //
+        // Distribute the patch correction vector back to the individual cells'
+        // FEEvaluation objects. 'false' indicates that DoFs shared beteween
+        // multiple cell are only written the first owning cell, not
+        // accumulated. The values of shared DoFs in subsequent cells will be
+        // set to 0.
+        patch_eval.distribute_patch_to_local(
+          ArrayView<const number>(local_correction), false);
+
+        // Add the corrections stored in the local FEEvaluation objects
+        // to the global solution vector 'dst'.
+        patch_eval.distribute_local_to_global(dst);
+      }
+  }
+
+
+} // namespace Operators
+
+
+namespace LaplaceSolver
+{
+  using namespace dealii;
+
+  /**
+   * Implements a Laplace solver using matrix-free techniques.
```

Comment:
I suggest to use a full sentence here
```suggestion
   * This class implements a Laplace solver using matrix-free techniques.
```